### PR TITLE
[Backport] [API] Allow only specific Colums to be updated on Issue (#9539)

### PR DESCRIPTION
**Pull request by 6543**
backports #9539

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

File: models/issue.go [hunk 1]
```diff
@@ -1730,30 +1743,24 @@ func SearchIssueIDsByKeyword(kw string, repoID int64, limit, start int) (int64,
...
+		"name", "is_closed", "content", "milestone_id", "priority",
+		"deadline_unix", "updated_unix", "closed_unix", "is_locked").
+		Update(issue); err != nil {
 		return err
 	}
+
 	if err := issue.neuterCrossReferences(sess); err != nil {
 		return err
 	}
-	if err := issue.loadPoster(sess); err != nil {
```
Comment (lunny) [inline models/issue.go]: Why delete this line? issue.Poster will be reference in lin 1757. We should ensure it loaded.

**Review by lunny — CHANGES_REQUESTED** [review]

File: models/issue.go (hunk 1 above)
Comment (lunny) [reply models/issue.go]: Maybe keep these lines to not change is better. So that reviewer could ignore the bother information.

**Comment by 6543** [comment]
@lunny done + #9583

**Comment by lunny** [comment]
@6543 see my comment on https://github.com/go-gitea/gitea/pull/9580#discussion_r362373273 , Why we need to move the `loadPoster` before `neuterCrossReferences`?

**Comment by 6543** [comment]
@lunny done see last commit (..message)

**Review by lunny — APPROVED** [review]

**Comment by 6543** [comment]
ready to merge :rocket:

**Pull request merged**